### PR TITLE
fix populatestate method of category model for com_contact

### PR DESCRIPTION
**Pull request by GeraintEdwards**
populatestate method of category model for com_contact was not using the user session data for the value of filter-search so pagination always failed to remember the value

fixes issue #9273

### Summary of Changes
Use $app->getUserStateFromRequest instead of $app->input->get to fetch the filter value

### CODE REVIEW

**Comment by carcam** [comment]
I have tested this item :white_check_mark: successfully on 08c483c80eaa2f05075ddfad0893aa7b37383e1a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16482">issues.joomla.org/tracker/joomla-cms/16482</a>.</sub>

File: components/com_contact/models/category.php [hunk 1]
```diff
@@ -248,7 +248,9 @@ protected function populateState($ordering = null, $direction = null)
 		$this->setState('list.start', $limitstart);
 
 		// Optional filter text
-		$this->setState('list.filter', $app->input->getString('filter-search'));
+                $itemid =$app->input->get('Itemid', 0, 'int');
```
Comment (Quy) [inline components/com_contact/models/category.php]: Add space after `=`

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 80008c51b79ef8f32f13eadd8ffc51d7579ade52<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16482">issues.joomla.org/tracker/joomla-cms/16482</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
@carcam can you please test again?

**Comment by Quy** [comment]
franz-wohlkoenig It is not necessary since it was only to add a space.

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Pull request merged**